### PR TITLE
 TW-909 Add API Governance Manager role

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 1]
```diff
@@ -89,6 +89,11 @@ Team roles offer high-level access control:
 | Enable public team profile | &#x2714; | &#x2714; | | | &#x2714; |
 | Manage a team's Private API Network&ast;&ast;&ast; | &#x2714; | | | | |
 
+There are additional specialized roles for Enterprise teams:
+
+* **API Network Manager** ([Enterprise plans only](https://www.postman.com/pricing)) - Manages a team's [Private API Network](/docs/collaborating-in-postman/adding-private-network/), including adding APIs and reviewing requests to add APIs.
+* **API Governance Manager** ([Enterprise plans only](https://www.postman.com/pricing)) - Manages API governance within a team.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:95]: Should we link "API governance" to a related page or section?

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! As discussed via Slack, there is some overlap with changes in the [tw-1126-folder-level-rbac-private-network](https://github.com/postmanlabs/postman-docs/compare/develop...tw-1126-folder-level-rbac-private-network) branch to be aware of when merging. Thanks!

File: src/pages/docs/collaborating-in-postman/roles-and-permissions.md [hunk 2]
```diff
@@ -89,6 +88,11 @@ Team roles offer high-level access control:
 | Enable public team profile | &#x2714; | &#x2714; | | | &#x2714; |
 | Manage a team's Private API Network&ast;&ast;&ast; | &#x2714; | | | | |
 
+There are additional specialized roles for [Enterprise](https://www.postman.com/pricing) teams:
+
+* **API Network Manager** - Manages a team's [Private API Network](/docs/collaborating-in-postman/adding-private-network/). To learn more, see [Network roles](#network-roles).
+* **API Governance Manager** - Manages [API governance](/docs/api-governance/api-governance-overview/) within a team.
```
Comment (savageCoder55) [inline src/pages/docs/collaborating-in-postman/roles-and-permissions.md:94]: @claireriley Should we consider adding more info here (Manages Governance rules, functions and Governance workspace groups within a team)?

**Review by savageCoder55 — APPROVED** [review]
One small comment. Nothing major.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**